lib/url.js

```diff
@@ -47,7 +47,8 @@ const {
   domainToUnicode,
   formatSymbol,
   pathToFileURL,
-  fileURLToPath
+  fileURLToPath,
+  urlToOptions
```

Comment:
nit: Add a trailing comma to make future PR diff cleaner?
```suggestion
  urlToOptions,
```

Reply:
A totally optional micro-nit not directly related to the change here, but the entries are sorted in ASCII order except for `fileURLToPath` so maybe move that under `domainToUnicode`?

Reply:
Both landed.